src/assets/javascripts/services.js

```diff
@@ -155,7 +155,7 @@ function redirect(url, type, initiator, forceRedirection) {
 			return url.href.replace(/^https?:\/{2}/, "yattee://")
 		}
 		case "freetube": {
-			return `freetube://https://youtu.be${url.pathname}${url.search}`.replace(/watch\?v=/, "")
+			return  'freetube://' + url.href.replace(/^https?:\/{2}/, "https//")
```

Comment:
why not just `url.href` why you added `url.href.replace(/^https?:\/{2}/, "https//")`?

Reply:
I was thinking for handling GTK urls but FreeTube should already handle it so I'll uldate it https://github.com/libredirect/browser_extension/issues/212#issuecomment-1116220925